apps/example/package.json

```diff
@@ -21,6 +21,7 @@
     "astro-portabletext": "^0.9.6",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "sanity": "^3.43.0",
```

Comment:
@stipsan , after removing `sanity` from the `peerDependencies` of the `@sanity/astro` package, the example `apps/*` can't resolve that dependency anymore. I'm not exactly sure why, but perhaps there's a gap in my understanding in regards to how npm workspaces work.